### PR TITLE
[Piksel] app token in <video> (uscourts.gov)

### PR DESCRIPTION
**Pull request by johnhawkinson**
Support alternative api token found as
  <video ... data-de-api-key="bd1863e1-a59e-11e3-bfa8-005056865f49" ...>
in addition to extraction from the flashvars JSON.

This supports the player as invoked from uscourts.gov, e.g. [http://www.uscourts.gov/cameras-courts/state-washington-vs-donald-j-trump-et-al](http://www.uscourts.gov/cameras-courts/state-washington-vs-donald-j-trump-et-al)

Add test case for same.

### CODE REVIEW

File: youtube_dl/extractor/piksel.py [hunk 1]
```diff
@@ -16,18 +16,32 @@
...
+                'id': 'nv60p12f',
+                'ext': 'mp4',
+                'title': 'فن الحياة  - الحلقة 1',
+                'description': 'احدث برامج الداعية الاسلامي " مصطفي حسني " فى رمضان 2016علي النهار نور',
+                'timestamp': 1465231790,
+                'upload_date': '20160606',
+            }
+        },
+        {
+            'url': 'http://www.uscourts.gov/cameras-courts/state-washington-vs-donald-j-trump-et-al',
```
Comment (dstftw) [inline youtube_dl/extractor/piksel.py]: All tests placed here must have URLs that match extractor's `_VALID_URL`. Tests for generic embeds should be placed into generic extractor.
```
PS C:\Dev\youtube-dl\master> py -3.5 .\test\test_all_urls.py
...F.............
======================================================================
FAIL: test_no_duplicates (__main__.TestAllURLsMatching)
----------------------------------------------------------------------
Traceback (most recent call last):
  File ".\test\test_all_urls.py", line 94, in test_no_duplicates
    self.assertTrue(ie.suitable(url), '%s should match URL %r' % (type(ie).__name__, url))
AssertionError: False is not true : PikselIE should match URL 'http://www.uscourts.gov/cameras-courts/state-washington-vs-donald-j-trump-et-al'

----------------------------------------------------------------------
Ran 17 tests in 8.066s

FAILED (failures=1)
```

Comment (johnhawkinson) [inline youtube_dl/extractor/piksel.py]: Would it be better to put this test in the generic extractor, or to change this test to the URL that gets resolved by the generic extractor, [https://player.piksel.com/v/v80kqp41](https://player.piksel.com/v/v80kqp41)?

Comment (dstftw) [inline youtube_dl/extractor/piksel.py]: I'd prefer the latter. Generic extractor is already too big.

Comment (johnhawkinson) [inline youtube_dl/extractor/piksel.py]: Done. Thanks.

**Comment by dstftw** [comment]
Thanks.

**Pull request merged**